.nf-core.yml

```diff
@@ -2,6 +2,7 @@ bump_version: null
 lint:
   files_exist:
     - conf/igenomes.config
+    - conf/igenomes_ignored.config
```

Comment:
is this correct, @mirpedrol, I thought we needed this even when not using igenomes

Reply:
It is correct, this file is to allow the use of `params.igenomes_ignore`, but if the igenomes is completely excluded from the pipeline, then we don't need any of these files.

Reply:
okay, thanks!